src/logic /operate.js

```diff
@@ -0,0 +1,20 @@
+import Big from 'big.js';
+
+const operate = (numberOne, numberTwo, operation) => {
+  let result;
+  if (operation === '+') {
+    result = Big(numberOne).plus(Big(numberTwo));
+  } else if (operation === '-') {
+    result = Big(numberOne).minus(Big(numberTwo));
+  } else if (operation === '/') {
+    result = Big(numberOne).div(Big(numberTwo));
+  } else if (operation === 'X') {
+    result = Big(numberOne).times(Big(numberTwo));
+  } else if (operation === '%') {
+    result = (Big(numberOne).times(Big(numberTwo))) / 100;
```

Comment:
Kindly use for example num1.mod(mun2) to convert a number into a percentage!!! Thus making use of the modifier method as pointed out in the previous review